client/src/scene/SpectatorUI.ts

```diff
@@ -104,147 +128,67 @@ export class SpectatorUI {
      * Remove the spectator voting section.
      */
     private removeTimedEvent() {
-        this.countdown.setText("");
-        this.alreadyVoted.setText("");
-
+        if (this.countdownTimer) {
+            clearInterval(this.countdownTimer);
+        }
+        this.header.setVisible(false);
+        this.countdown.setVisible(false);
+        this.alreadyVoted.setVisible(false);
+        this.hideButtons();
         this.cookieTracker.deleteCookie("hasVoted");
```

Comment:
this was a mistake on my part. Can you move this `deleteCookie` line to be inside the hideVotingSequence listener?
https://github.com/ECE493G14Capstone/Capstone/blob/97882f15d883f26c28ddb3565b55b3e00667da1f/client/src/scene/SpectatorUI.ts#L110

Atm, whenever you refresh, the cookies are wiped which should not happen. They should only be wiped if they receoved an event from the server telling it to close the voting sequence.

Reply:
Resolved in commit [296ffd7](https://github.com/ECE493G14Capstone/Capstone/pull/77/commits/296ffd73e7d687187c9fe6cef5033c4ba46c6146).